### DISCUSSION TITLE
ESP-01S not broadcasting SSID #Solved

### DISCUSSION THREAD

**Discussion by luiscesjr** [luc-github/ESP3D#839] [Questions]
I'm having a bit of trouble setting up my ESP. Some background on the history of issues:

I was using Octopi, but suddenly it stopped working on my router. New octopi images, new boards, tried everything, it did not work, so I had to reflash my printer fw to be able to read from my SKR 1.4 turbo board sdcard, which was not enabled before.
I found my ESP 01S that came with my board, and thought about adding it to have a bit of the octopi's features, and after the initial setup all was alright.

The problems started when I decided to go from AP mode to connect to my router, the same octopi problem happened, and I could not access the ESP anymore.  I decided to then erase the ESP flash and install again the ESP3D onto it, but somehow it's not working anymore, it just won't broadcast the initial wifi.

I am able to erase it and install again, tried different ESP3D versions, but results are the same. The monitor on Vscode gives out nothing really, so I can't know if it's having some issue.
It stays like this:
![image](https://user-images.githubusercontent.com/7636427/207071785-aaba0f61-55f9-4190-aec9-e42cf480f5f3.png)

Any ideas what I could start from to look for a solution?  Thanks!

**Comment by luiscesjr** [comment]
So after trying to esp flasher and the vscode hundreds of times, I decided to try the Arduino IDE, somehow there it started working again on the latest fw of esp3d. I have no idea why there it finally started working, but it did. So if anyone ever face the same problem...

I started by uploading a blink led to check the board was functional, and then dug through the configs to get my board up and running again.

**Reply by luc-github** [reply]
if after flashing an esp8266 there is no brodcast neither serial output it usually:
1 - flash size defined is wrong, esp8266 does not raise error if you try to flash an 1MB flash but you set 4MB flash
2- rarely the Flash mode must be changed on esp8266 from dio to another
3 - on esp8266-01 be sure the pins CH_PD / RESET / GPIO 0 et eventually GPIO 2 must be high after flash https://github.com/luc-github/ESP3D/wiki/Hardware-connection#esp-01
4 - depending on ftdi adapter a restart may be necessary after flash to match the point 3
